build.proj

```diff
@@ -16,15 +16,7 @@
     <BuildDependsOn Condition="'$(SignPackages)'=='true'">$(BuildDependsOn);SignPackages</BuildDependsOn>
   </PropertyGroup>
 
-  <Import Project="$(ToolsDir)\VersionTools.targets" Condition="Exists('$(ToolsDir)\VersionTools.targets')" />
-
-   <Target Name="Restore">
-    <ItemGroup>
-      <_RestoreProjects Include="external/**/*.depproj" />
-    </ItemGroup>
-
-    <MSBuild Projects="@(_RestoreProjects)" />
-  </Target>
+  <Target Name="Restore"/>
```

Comment:
You now also need this change: https://github.com/dotnet/corefx/pull/34168/files#diff-991506ab3bfb2ea4767ed342c4f95726R5

Reply:
It disables calling directly into the NuGet.targets restore target if the repository build project does not use NuGet restore.

Reply:
https://github.com/dotnet/corefx/issues/34164